scipy/stats/tests/test_distributions.py

```diff
@@ -3258,6 +3258,26 @@ def test_ncx2_tails_pdf():
     assert_(np.isneginf(logval).all())
 
 
+@pytest.mark.parametrize('method, expected', [
+    ('cdf', np.array([1.72115630e-04, 2.74102033e-05])),
+    ('pdf', np.array([0.00078975, 0.00013013])),
+    ('logpdf', np.array([-7.14378973, -8.94700192]))
```

Comment:
Please comment on where are these numbers from (imagine that ten years down the line, someone stares at these lines; your comment should help them check the numbers)